config/routes.rb

```diff
@@ -13,6 +13,8 @@
   get 'posts/new', to: 'posts#new', as: 'new_post' # new
   post '/posts/new', to: 'posts#create', as: 'create_post' # create
   get '/users/:user_id/posts/:id', to: 'posts#show', as: 'post' # show
+  delete '/users/:user_id/posts/:id', to: 'posts#destroy', as: 'delete_post' # delete
+  delete '/users/:user_id/posts/:id/:comment', to: 'posts#destroy_comment', as: 'destroy_comment'
   post '/users/:user_id/posts/:id', to: 'posts#create_comment'
   post '/users/:user_id/posts/:id/like', to: 'posts#like', as: 'like_post'
```

Comment:
- [ ] Please edit these routes because comments suppose to have `comments#new`, `comments#create`, and `comments#destroy` routes and likes suppose to have `likes#create` route.
[Hint]: You can do all routes like that:
```ruby
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show, :new, :destroy] do
      resources :comments, only: [:new, :create, :destroy]
      resources :likes, only: [:create]
    end
  end

root "users#index"
```